indexing-hadoop/src/main/java/io/druid/indexer/updater/HadoopConverterJob.java

```diff
@@ -140,11 +140,17 @@ public static Path getTaskPath(JobID jobID, TaskAttemptID taskAttemptID, Path wo
     return new Path(getJobPath(jobID, workingDirectory), taskAttemptID.toString());
   }
 
+  public static Path getJobClassPathDir(String jobName, Path workingDirectory) throws IOException
+  {
+    return new Path(workingDirectory, jobName.replace(":", ""));
+  }
+
   public static void cleanup(Job job) throws IOException
   {
     final Path jobDir = getJobPath(job.getJobID(), job.getWorkingDirectory());
     final FileSystem fs = jobDir.getFileSystem(job.getConfiguration());
     fs.delete(jobDir, true);
+    fs.delete(getJobClassPathDir(job.getJobName(), job.getWorkingDirectory()), true);
```

Comment:
I think we need to do this for cleanup for other Hadoop Jobs as well, would like to do it in a separate PR. Opening a github issue